checkout/src/main/java/com/checkout/validation/validator/CvvValidator.kt

```diff
@@ -41,9 +59,17 @@ internal class CvvValidator : Validator<CvvValidationRequest, Unit> {
                 ValidationError.CVV_CONTAINS_NON_DIGITS,
                 "CVV should contain only digits"
             )
+            isEager && cvv.length > cvvLength.last() -> throw ValidationError(
+                ValidationError.CVV_INVALID_LENGTH,
+                "CVV length is invalid, it length should be $cvvLength"
+            )
+            isEager && isPartialCvvEntry(cvv, cvvLength) -> throw ValidationError(
+                ValidationError.CVV_INCOMPLETE_LENGTH,
+                "CVV length is incomplete, it should be $cvvLength"
+            )
```

Comment:
After those changes become hard readable.
Probably would be better to create a separate function i.e. `provideEagerCvvValidationResult(cvv: String, cvvLength: Set<Int>)`.

Reply:
An empty string probably should be valid for the eager check.
Please clarify it.